libs/auth/src/java_card.ts

```diff
@@ -112,14 +146,17 @@ export class JavaCard implements Card {
   private readonly cardReader: CardReader;
   private cardStatus: CardStatus;
   private readonly jurisdiction: string;
+  private readonly vxAdminParams?: VxAdminParams;
   private readonly vxCertAuthorityCertPath: string;
 
   constructor(input: {
     jurisdiction: string;
+    vxAdminParams?: VxAdminParams;
```

Comment:
If these are only used in `program`, could they be passed in directly there? Also maybe name them something related to programming rather than VxAdmin?

Reply:
Yeah I went back and forth on whether to name this unit in terms of function or machine but agree that function makes more sense!

In terms of passing params to `program` (and `unprogram`), that would require me to change the general `Card` interface, leaking aspects of and making it more specific to the Java Card + X.509 cert implementation. While I know we'll be getting rid of the memory card implementation of the `Card` interface soon, leaving only one implementation, I'd still like to keep the `Card` interface general. The params are also pretty low-level and not something that I'd want engineers to have to think about when calling `program`.